### ISSUE TITLE
securityUtility createSSLCertificate does not accept or handle multiple --extInfo options

### ISSUE THREAD

**Issue by ibmssou** [OpenLiberty/open-liberty#9767]
macOS is strictly enforcing new certificate requirements (https://support.apple.com/en-us/HT210176), requiring both SAN (SubjectAlternateName) and EKU (ExtendedKeyUsage) values set in the SSL certificate.

By default, securityUtility automatically sets SAN, but if --extInfo is used, SAN is not automatically set and only the value passed to extInfo is used in SSL cert generation:

```
# securityUtility.bat createSSLCertificate --server=webServer --password=default --validity=825 \
--extInfo="ExtendedKeyUsage=serverAuth"

# ikeycmd -cert -details -db key.p12 -label default -pw default
Label: default
Key Size: 2048
Version: X509 V3
Serial Number: 3D 4E C8 8D
Issued by: CN=9.211.109.222, OU=webServer, O=ibm, C=us
Subject: CN=9.211.109.222, OU=webServer, O=ibm, C=us
Valid: From: Wednesday, November 13, 2019 9:12:30 PM MST To: Tuesday, February 15, 2022 9:12:30 PM MST
Fingerprint:
    SHA1: A7:8D:E5:5E:7E:95:33:2C:2A:A9:AF:7D:D3:FE:4B:69:68:58:6F:68
    SHA256: 8C:1C:2A:9C:8E:38:C8:16:4B:BC:13:0C:F9:2E:4F:26:39:AC:C9:67:20:80:8C:8F:3E:A3:CA:A7:DD:70:27:DB
    HPKP: BklPbHcFWIqnpwXv2Jo8B36jvSVoyj/yzS5sACxilIY=


Extensions:
  - SubjectKeyIdentifier: ObjectId: 2.5.29.14 Criticality=false
SubjectKeyIdentifier [
KeyIdentifier [
0000: 7e b4 91 20 09 e3 1c 90  25 89 57 b8 94 b0 47 c6  ..........W...G.
0010: bb b5 ff a3                                        ....
]
]

  - ExtKeyUsage: serverAuth
Signature Algorithm: SHA256withRSA (1.2.840.113549.1.1.11)
Trust Status: enabled
```

Notice the cert does not have SubjectAlternativeName set.  According to the securityUtility documentation, the --extInfo parameter is passed directly to the keytool command:

> --extInfo
> Specifies any X.509 certificate extension information that it uses to create the default certificate. By default, the Subject Alternative Name extension is added to the certificate with the host name of the server. If the --extInfo option is used, the value is passed directly to the keytool command. Use the syntax of the --ext flag on the keytool command.

I've tried many different variations of trying to pass multiple values to --extInfo.  Comma separated doesn't work:

```
# securityUtility createSSLCertificate --server=webServer --password=default --validity=825 \
--extInfo="ExtendedKeyUsage=serverAuth,san=DNS:cohiba5.storage.tucson.ibm.com"
Creating keystore .../key.p12
Unable to create default SSL certificate:
keytool error: java.lang.Exception: Unknown extendedkeyUsage type: san=DNS:cohiba5.storage.tucson.ibm.com
Error: java.security.cert.CertificateException: keytool error: java.lang.Exception: Unknown extendedkeyUsage type: san=DNS:cohiba5.storage.tucson.ibm.com
```

And neither does semi-colon:

```
# securityUtility createSSLCertificate --server=webServer --password=default --validity=825 \
--extInfo="ExtendedKeyUsage=serverAuth;san=DNS:cohiba5.storage.tucson.ibm.com"
Creating keystore .../key.p12
Unable to create default SSL certificate:
keytool error: java.lang.Exception: Unknown extendedkeyUsage type: serverAuth;san=DNS:cohiba5.storage.tucson.ibm.com
Error: java.security.cert.CertificateException: keytool error: java.lang.Exception: Unknown extendedkeyUsage type: serverAuth;san=DNS:cohiba5.storage.tucson.ibm.com
```

Or spaces:

```
# securityUtility createSSLCertificate --server=webServer --password=default --validity=825 \
--extInfo="ExtendedKeyUsage=serverAuth san=DNS:cohiba5.storage.tucson.ibm.com"
Creating keystore .../key.p12
Unable to create default SSL certificate:
keytool error: java.lang.Exception: Unknown extendedkeyUsage type: serverAuth san=DNS:cohiba5.storage.tucson.ibm.com
Error: java.security.cert.CertificateException: keytool error: java.lang.Exception: Unknown extendedkeyUsage type: serverAuth san=DNS:cohiba5.storage.tucson.ibm.com
```

I spoke with Alaine Demyers a few weeks back in Slack and he took a look at the code and saw that the code that handles the --extInfo parameter does not handle multiple --extInfo flags at all, which is a problem.

We currently generate our certs with keytool directly as a workaround, but it would be ideal if securityUtility could handle multiple --extInfo values properly.

**Comment by acdemyers** [comment]
I will fix the createSSLCertificate utility to only not include the default SAN if a SAN is passed into the --extInfo flag.   Right now anytime the --extInfo flag is used the default SAN information is not put in the certificate.  Keep in mind the command does not include a default SAN if the hostname for some reason starts with digits.  

The code will also be updated to handle multiple extInfo flags to make it more similar to keytool use of the ext flag.   

I tried the example above were the information was separated by a comma, semi colon or a space with the  keytool command.   The semi-colon execute without a failure but keytool does not seem to do anything with the part after the semi colon.  Keytool gives an error when using a comma or space.  So I will not support that kind of syntax.

**Comment by acdemyers** [comment]
Fixed in 19.0.0.12
